backends/module.mk

```diff
@@ -74,9 +74,11 @@ MODULE_OBJS += \
 
 # SDL 1.3 removed audio CD support
 ifndef USE_SDL13
+ifndef USE_SDL2
```

Comment:
It did not even try to use it because starting with SDL1.3 (a prerelease used by our PS3 port) we didn't include it anymore. Instead it used the default audio cd implementation (i.e. emulation only). I guarded the implementation with some #ifs now so even trying to including the code which would try to use it will make it fail at unit compile time.